### DISCUSSION TITLE
DB table prefix as a command line argument

### DISCUSSION THREAD

**Discussion by ioannist** [subsquid/squid-sdk#173] [Ideas]
Would be nice to be able to pass a DB table name prefix as a parameter. This would allow archiving multiple parachains on the same DB/schema. Without a prefix, the tables will clash.

**Comment by eldargab** [comment]
What are the cons of just using schemas? After all that's exactly what schemas are made for!?

**Reply by milmi172-hub** [reply]
Great

**Comment by ioannist** [comment]
You are right. In our case, we thought we were limited by one schema (supabase) but there seems to be a way around it after all.
